tools/RAiDER/statsPlot.py

```diff
@@ -494,19 +547,24 @@ def __call__(self, gridarr, plottype, workdir='./', drawgridlines=False, colorba
         import cartopy.io.img_tiles as cimgt
         import cartopy.crs as ccrs
         from cartopy.mpl.ticker import LongitudeFormatter, LatitudeFormatter
+        import cartopy.feature as cfeature
         import matplotlib.ticker as mticker
 
         # If specified workdir doesn't exist, create it
         if not os.path.exists(workdir):
             os.mkdir(workdir)
 
-        basemap   = cimgt.Stamen('terrain-background')
-        # convert basemap to black-and-white mode
-        basemap.desired_tile_form = 'L'
-        fig, axes = plt.subplots(subplot_kw={'projection': basemap.crs})
-        axes.set_extent(self.plotbbox, ccrs.Geodetic())
-        axes.add_image(basemap, resValue, cmap='gray')
-        axes.coastlines()
+        fig, axes = plt.subplots(subplot_kw={'projection': ccrs.PlateCarree()})
+        # by default set background to white
+        axes.add_feature(cfeature.NaturalEarthFeature('physical', 'land', '50m', facecolor='white'), zorder=0)
+        # reset in case bounds too large
+        try:
+            axes.set_extent(self.plotbbox, ccrs.PlateCarree())
+        except:
```

Comment:
Should either specify the error type or explicitly catch all exceptions with
```suggestion
        except Exception:
```